### PR TITLE
bench-tps/net sanity: add ability to check for unexpected extra nodes

### LINKED ISSUE #712: master.testnet.solana.com node count is too damn high!
After resetting m.t.s.c, the node count should be exactly 50 (unless somebody is running extra validator nodes attached to m.t.s.c -- possible but unlikely at this point).

After adding 17d927ac74ebb24d7fc5ce07c70ed5f9bc1a008c, it shows a node count of 80 or other numbers not 50.

It's possible we have some kind of crdt table issue.

STR: Look at the output from the latest build at https://buildkite.com/solana-labs/testnet-deploy

cc: #693 
cc: #695

### PR DESCRIPTION
**Pull request by mvines**
Fixes #712

Or rather, if this doesn't fix #712 then it'll turn the testnet red and we'll know.

### CODE REVIEW

File: net/remote/remote-sanity.sh [hunk 1]
```diff
@@ -89,7 +93,18 @@ echo "+++ $entrypointIp: node count ($numNodes expected)"
 (
   set -x
   $solana_keygen -o "$client_id"
-  $solana_bench_tps --network "$entrypointIp:8001" --identity "$client_id" --num-nodes "$numNodes" --converge-only
+
+  maybeRejectExtaNodes=
```
Comment (mvines) [inline net/remote/remote-sanity.sh]: Note to self: add the missing r here...

**Comment by rob-solana** [comment]
what problem is this solving?

**Comment by rob-solana** [comment]
perhaps we should rename bench-tps to something like "test-client", since it's becoming a dumping ground for network test-type functionality

**Comment by mvines** [comment]
> what problem is this solving?

It's a very complex way of turning this [1] < into a = sometimes.  On a private net we really want = to detect if extra junk snuck into CRDT.  On a public net, somebody else may legitimately be on the net so it must be < (ensuring at least our nodes are *likely* present)

[1] https://github.com/solana-labs/solana/blob/master/src/bin/bench-tps.rs#L513


> perhaps we should rename bench-tps to something like "test-client",

Yeah not a bad idea IMO

**Pull request merged**